pkg/util/testing/wrappers.go

```diff
@@ -110,6 +110,16 @@ func (j *JobWrapper) Request(r corev1.ResourceName, v string) *JobWrapper {
 	return j
 }
 
+func (j *JobWrapper) Image(index int, name string, image string, args []string) *JobWrapper {
```

Comment:
I would hard code it to zero. I don't expect us to ever need more than one image in a Job for kueue tests.

```suggestion
func (j *JobWrapper) Image(name string, image string, args []string) *JobWrapper {
```

Reply:
Done.